envi/__init__.py

```diff
@@ -36,7 +36,7 @@
 ARCH_PPCVLE      = 15 << 16
 ARCH_PPC_D       = 16 << 16
 ARCH_MCS51       = 17 << 16
-ARCH_RXV2        = 18 << 16
+ARCH_RX          = 18 << 16
```

Comment:
As a backwards compatibility thing, can we leave this as ARCH_RXV2? Just in case anyone is using this module export.